packages/lib-classifier/README.md

```diff
@@ -1,6 +1,41 @@
 # Zooniverse Classifier
 
-A standalone library for the Zooniverse project classifier, including state management and UI component views.
+A standalone library for the Zooniverse Project Classifier. The `Classifier` is comprised of three major pieces - the [SubjectViewer](/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/README.md), the [Tasks](/packages/lib-classifier/src/plugins/tasks/README.md), and the [Store](/packages/lib-classifier/src/store/README.md).
+
+```mermaid
+flowchart TD
+    Classifier-->SubjectViewer
+    Classifier-->TaskArea
+    Classifier-->RootStore
```

Comment:
In the rest of the documentation, you use the term `Tasks` and `Store`, but in the mermaid charts (in this block, ad the block below under 'Classifier Render Process'), you use `TaskArea` and `RootStore`. I understand the intent here, but I'm not sure if a new dev would appreciate the nuances between different terms.

I'm not sure if it's better to add a note ("PS: the TaskArea is the component where all Tasks are displayed on the Classifier, and the RootStore is the base unit/root/node for the entire Store system") or to relabel the mermaid charts. Either way, this is _not_ a major note.